dumper/app/src/test/resources/ranger/services.success.jsonl

```diff
@@ -1,2 +1,2 @@
-{"configs":{"fs.default.name":"hdfs://hadoop-m:8020","hadoop.security.authentication":"Simple","hadoop.security.authorization":"No","password":"*****","username":"admin"},"createdBy":"Admin","description":"Hadoop hdfs service","displayName":null,"guid":"4c2780ee-e244-4ac3-b21a-3660e6895980","id":1,"name":"hadoop-dataproc","policyUpdateTime":"2024-07-26T17:58:21Z","policyVersion":9,"tagService":null,"tagUpdateTime":"2024-07-26T17:58:22Z","tagVersion":1,"type":"hdfs","updatedBy":"Admin","version":1,"createDate":"2024-05-27T07:34:42Z","enabled":true,"updateDate":"2024-05-27T07:34:42Z"}
-{"configs":{"jdbc.driverClassName":"org.apache.hive.jdbc.HiveDriver","jdbc.url":"jdbc:mysql://localhost","password":"*****","username":"admin"},"createdBy":"Admin","description":"Hadoop HIVE service","displayName":null,"guid":"1fe46519-233a-4780-be20-a7eb180f1ca6","id":2,"name":"hive-dataproc","policyUpdateTime":"2024-07-26T17:58:21Z","policyVersion":6,"tagService":null,"tagUpdateTime":"2024-07-26T17:58:22Z","tagVersion":1,"type":"hive","updatedBy":"Admin","version":1,"createDate":"2024-05-27T07:34:42Z","enabled":true,"updateDate":"2024-05-27T07:34:42Z"}
+{"configs":{"fs.default.name":"hdfs://hadoop-m:8020","hadoop.security.authentication":"Simple","hadoop.security.authorization":"No","password":"*****","username":"admin"},"createdBy":"Admin","description":"Hadoop hdfs service","displayName":null,"guid":"4c2780ee-e244-4ac3-b21a-3660e6895980","id":1,"name":"hadoop-dataproc","policyUpdateTime":"2024-07-26T17:58:21Z","policyVersion":9,"tagService":null,"tagUpdateTime":"2024-07-26T17:58:22Z","tagVersion":1,"type":"hdfs","updatedBy":"Admin","version":1,"createDate":1716795282000,"enabled":true,"updateDate":1716795282000}
+{"configs":{"jdbc.driverClassName":"org.apache.hive.jdbc.HiveDriver","jdbc.url":"jdbc:mysql://localhost","password":"*****","username":"admin"},"createdBy":"Admin","description":"Hadoop HIVE service","displayName":null,"guid":"1fe46519-233a-4780-be20-a7eb180f1ca6","id":2,"name":"hive-dataproc","policyUpdateTime":"2024-07-26T17:58:21Z","policyVersion":6,"tagService":null,"tagUpdateTime":"2024-07-26T17:58:22Z","tagVersion":1,"type":"hive","updatedBy":"Admin","version":1,"createDate":1716795282000,"enabled":true,"updateDate":1716795282000}
```

Comment:
Are `tagUpdateTime` and `policyUpdateTime` attributes that also contain timestamps unaffected?